core/src/mindustry/mod/ClassMap.java

```diff
@@ -222,7 +224,7 @@
         classes.put("DriverState", mindustry.world.blocks.distribution.MassDriver.DriverState.class);
         classes.put("MassDriverBuild", mindustry.world.blocks.distribution.MassDriver.MassDriverBuild.class);
         classes.put("OverflowDuct", mindustry.world.blocks.distribution.OverflowDuct.class);
-        classes.put("DuctRouterBuild", mindustry.world.blocks.distribution.OverflowDuct.DuctRouterBuild.class);
+        classes.put("DuctRouterBuild", OverflowDuctBuild.class);
```

Comment:
I don't know why the auto generation of ClassMap decided import instead of using the classpath.